storage/innobase/btr/btr0sea.cc

```diff
@@ -218,8 +218,7 @@ ATTRIBUTE_COLD static void btr_search_lazy_free(dict_index_t *index)
   table->autoinc_mutex.wr_unlock();
 }
 
-/** Disable the adaptive hash search system and empty the index. */
```

Comment:
Just thinking it would be helpful to others reading the code to have the comment, even though AHI is generally disabled in MariaDB